graph_shortest_path/routing.py

```diff
@@ -45,16 +50,33 @@ def find_vertex(self, value):
         pass
```

Comment:
You're checking for the value of the vertex...how else will the program identify the vertex? 
You don't need the pass statement once you put in code